code/_onclick/click_catcher.dm

```diff
@@ -1,127 +1,62 @@
-/*
-	A click catcher is an invisible object on the client screen which draws below pretty much everything else, including turfs.
```

Comment:
why have you removed all this documentation and helper procs?

Reply:
> why have you removed all this documentation and helper procs?


I believe that the documentation is outdated for this code and in addition it is not needed. Since here she performs the most basic mechanics. It is generally better not to make changes in it.